app/bundles/LeadBundle/Entity/LeadRepository.php

```diff
@@ -917,6 +917,35 @@ protected function addSearchCommandWhereClause($q, $filter): array
                 );
                 $returnParameter = true;
                 break;
+            case $this->translator->trans('mautic.lead.lead.searchcommand.dnc'):
+            case $this->translator->trans('mautic.lead.lead.searchcommand.dnc', [], null, 'en_US'):
+                $anyKeyword = $this->translator->trans('mautic.lead.lead.searchcommand.dnc.any');
+                if ($string === $anyKeyword) {
```

Comment:
The way it works now is that we are filtering for the filter keyword in English as well as in the local language. This way the `mautic.lead.lead.searchcommand.dnc.any` translation works only for English. Let's build it the same way as other search commands that allow also for the local language keyword.